cypress/page/payment-step.page.ts

```diff
@@ -0,0 +1,23 @@
+class  PaymentStepPage {
+    private payButton: string;
+    private confirmOrder: string;
+
+    constructor() {
+        this.payButton = "a.bankwire";
+        this.confirmOrder = "#cart_navigation>button>span";
+    }
+
+    public confirmOrderm(): void {
+        cy.get(this.confirmOrder).click();
+    }
+
+    public bankwirepay(): void {
+        cy.get(this.payButton).click()
+    }
+
+    public confirmMessage(message: string) {
+        cy.get("#center_column > div > p > strong").should("have.text", message);
+        return this;
```

Comment:
Yo can remove this return

Reply:
done @danielgalvis98

Reply:
Are you using this return? Try again removing it (and rmember adding : void as return type), I think some of the previous tests were failling due to problems on the server